css/components/link.css

```diff
@@ -8,3 +8,37 @@
   margin-right: var(--link-icon-margin);
 }
 
+.link--button-style .link-wrapper {
+  padding:
+  var(--button-padding-vertical) var(--button-padding-horizontal)
+  var(--button-padding-vertical) var(--button-padding-horizontal);
+  cursor: pointer;
```

Comment:
We don't need `cursor:pointer` as this is a link, it will automatically have this.